### ISSUE TITLE
Add tests for wcsncasecmp()

### ISSUE THREAD

**Issue by anton-bondarev** [embox/embox#2349]

**Comment by VirenderNayal** [comment]
Hello,
I want to work on this issue.
I am new to this repository, can you help me to figure out.

**Comment by anton-bondarev** [comment]
@VirenderNayal I've assigned it to you.

You should start with the build and run Embox on QEMU See [Quick-Start](https://github.com/embox/embox/wiki/Quick-Start)
After this, you have to add tests for wcsncasecmp() debug it and create PR with your cahnges

**Comment by VirenderNayal** [comment]
Thanks for assigning it to me..
I have followed the quick-start guide, after the `make` process

I ran this command :
`sudo ./scripts/qemu/auto_qemu`

It is giving an error:

![image](https://user-images.githubusercontent.com/72212823/108554093-9e2c1d80-7319-11eb-81aa-e3da19aac577.png)

I am using Endeavour OS (Arch Based)

**Comment by alexkalmuk** [comment]
@VirenderNayal Can you please tell your QEMU version?

**Comment by VirenderNayal** [comment]
@alexkalmuk I have tried to check, see this:

![image](https://user-images.githubusercontent.com/72212823/108602328-6aa7cc80-73c7-11eb-98b3-ea260ae1b27d.png)

**Comment by alexkalmuk** [comment]
@VirenderNayal Sorry for the delay, could you please tell me which template you build? `x86/qemu`? Also, what is your gcc version?

```
$ gcc --version
gcc (Ubuntu 9.3.0-17ubuntu1~20.04) 9.3.0
```

Also, show, please

```
$ file ./build/base/bin/embox
./build/base/bin/embox: ELF 32-bit LSB executable, Intel 80386, version 1 (SYSV), statically linked, with debug_info, not stripped
```

**Comment by VirenderNayal** [comment]
@alexkalmuk Here are the details:
I used x86/qemu

![image](https://user-images.githubusercontent.com/72212823/109743853-5e0d4a80-7bf7-11eb-9326-9b44209f9416.png)

**Comment by alexkalmuk** [comment]
@VirenderNayal I built Embox with gcc version 10.2-0 and run with qemu-5.2.0, so I still have no idea what's going on. Can you please attach your `build/base/bin/embox` here?

**Comment by VirenderNayal** [comment]
@alexkalmuk 
`build/base/bin/embox` : 

![image](https://user-images.githubusercontent.com/72212823/110200519-747c0600-7e84-11eb-8d08-c6418addb249.png)


I have done all the processes again and ran the command and :
`./scripts/qemu/auto_qemu`

![image](https://user-images.githubusercontent.com/72212823/110200597-cde43500-7e84-11eb-87a4-217ccbfdb046.png)


I think the problem is with these:

`VT is not supported by CPU
dmesg: read kernal buffer failed: Operation not permitted`

**Comment by alexkalmuk** [comment]
@VirenderNayal I meant can you please attach your file build/base/bin/embox so I could launch it too?

**Comment by alexkalmuk** [comment]
Can you please also try to launch it with sudo?

```
$ sudo ./scripts/qemu/auto_qemu
```

**Comment by VirenderNayal** [comment]
I ran with sudo no change.

**Comment by alexkalmuk** [comment]
Ok, then can you please attach your file here? (build/base/bin/embox)

**Comment by VirenderNayal** [comment]
I am unable to attach my build/base/bin/embox
It says format not supported

**Comment by alexkalmuk** [comment]
Ah. ok

**Comment by VirenderNayal** [comment]
I got this after i ran with sudo

![image](https://user-images.githubusercontent.com/72212823/110200928-6d55f780-7e86-11eb-80d3-80a683d099ad.png)

**Comment by alexkalmuk** [comment]
Can you add your file to for example https://filebin.net/ and provide me a link?

**Comment by VirenderNayal** [comment]
Here (build/base/bin/embox):

https://filebin.net/5wvs93u9nnukop6o

**Comment by alexkalmuk** [comment]
Thanks, it reproduced.  I will go back as soon as we find a solution.

**Comment by VirenderNayal** [comment]
okay thanks @alexkalmuk

**Comment by alexkalmuk** [comment]
btw could you show

```
$ ld --version
```

**Comment by VirenderNayal** [comment]
Yeah Sure.

![image](https://user-images.githubusercontent.com/72212823/110202628-08070400-7e90-11eb-99eb-94c810789fc3.png)

**Comment by niklvdanya** [comment]
Hi, I would like to get assigned.

**Comment by anton-bondarev** [comment]
Hi @niklvdanya , I've assigend it with you

**Comment by Khokholeva** [comment]
Hello, I would like to work on this issue

**Comment by niklvdanya** [comment]
Cloded #2959

**Comment by anton-bondarev** [comment]
Hello @Khokholeva
Today some tests have already been added. It's better if you choose another "googd first issue" (I've added some more now), but you can also expand existed test cases or add your implementation.

**Comment by anton-bondarev** [comment]
Closed by #2959 #2966 #2967
